<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="Shadow" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>605</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#Shadow{
background-color: rgb(56, 58, 89);
color: rgb(220, 220, 220);
}</string>
   </property>
   <widget class="QLabel" name="form">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>381</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 255);
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="side_lbl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>361</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#side_lbl{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 			rgba(85, 98, 112, 255), stop:1 rgba(255, 107, 107, 255));
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="header">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;</string>
    </property>
    <property name="text">
     <string>Login To you Facebook Account</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameField">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:2px solid rgb(255, 255, 255);
border-bottom-color:  rgba(46, 82, 101, 200);
color: rgb(0, 0, 0);
padding-bottom: 2px;</string>
    </property>
    <property name="placeholderText">
     <string>Your Email OR Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordField">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:2px solid rgb(255, 255, 255);
border-bottom-color:  rgba(46, 82, 101, 200);
color: rgb(0, 0, 0);
padding-bottom: 2px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="headlessField">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left:10px;</string>
    </property>
    <property name="text">
     <string>Headless 
  Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>400</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#login{
	background-color: rgba(85, 98, 112, 255);
	color: rgba(255, 255, 255, 200);
	border-radius:5px;
}
QPushButton#login:pressed{
	padding-left:5px;
	padding-top: 5px;
	background-color: rgba(255, 107, 107, 255);
	background-position:calc(100% - 10px)center;
}
QPushButton#login:hover{
	background-color: rgba(255, 107, 107, 255);
}</string>
    </property>
    <property name="text">
     <string>Proceed</string>
    </property>
   </widget>
   <widget class="QLabel" name="Tittle">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yrsa Medium</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Yegon Bot</string>
    </property>
   </widget>
   <widget class="QLabel" name="disclaimer">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="swords">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>331</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>221</pointsize>
     </font>
    </property>
    <property name="text">
     <string>âš”</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="threadsField">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>280</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:2px solid rgb(255, 255, 255);
border-bottom-color:  rgba(46, 82, 101, 200);
color: rgb(0, 0, 0);
padding-bottom: 2px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Threads Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorsField">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Z003</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font:12pt;
color: Red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="scrollsField">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>340</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:2px solid rgb(255, 255, 255);
border-bottom-color:  rgba(46, 82, 101, 200);
color: rgb(0, 0, 0);
padding-bottom: 2px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Scroll Count</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
